applications/openshift/logging/audit_log_forwarding_uses_tls/rule.yml

```diff
@@ -0,0 +1,54 @@
+prodtype: ocp4
+
+title: Ensure that Audit Log Forwarding Uses TLS
+
+description: |-
+  <p>
+  OpenShift audit works at the API server level, logging all requests coming to the server.
+  Audit is on by default and the best practice is to ship audit logs off the cluster for retention
+  using a secure protocol.
+  </p>
+  <p>
+  The cluster-logging-operator is able to do this with the <pre>ClusterLogForwarders</pre> resource.
+  The forementioned resource can be configured to logs to different third party systems.
+  For more information on this, please reference the official documentation:
+  {{{ weblink(link="https://docs.openshift.com/container-platform/latest/logging/cluster-logging-external.html") }}}
+  </p>
+
+rationale: |-
+  It is necessary to ensure that any configured output uses the TLS protocol to receive
+  logs in order to ensure the confidentiality, integrity and authenticity of the logs.
+
+references:
+  nist: AU-9,AU-9(2),AU-9(3),AU-10
+
+identifiers:
+  cce@ocp4: CCE-90688-3
```

Comment:
Wait a sec, where does the CCE come from? Did you maybe forget to remove a line from the CCE list?

Reply:
sure did. Let me fix this